core/models.py

```diff
@@ -111,11 +111,15 @@ class Group(models.Model):
 
     class Meta:
         verbose_name = 'Grupo'
+    CATEGORIES = [
+
+    ]
 
     group_name = models.TextField(verbose_name='Nombre grupo')
+    category = models.CharField('Tipo', choices=CATEGORIES, max_length=32)
```

Comment:
ok, tambien en forms se llama a fields = category ,tambien lo quito del medio?

Lo pytest dan error entiendo porque desde views , invoca el form que pide los category fields?

collected 29 items / 2 errors

=================================== ERRORS ====================================
_________________ ERROR collecting webapp/tests/test_forms.py _________________
webapp\tests\test_forms.py:6: in <module>
    from webapp import forms
webapp\forms.py:58: in <module>
    class GroupEdit(forms.ModelForm):
.venv\lib\site-packages\django\forms\models.py:240: in __new__
    "needs updating." % name
E   django.core.exceptions.ImproperlyConfigured: Creating a ModelForm without either the 'fields' attribute or the 'excl
ude' attribute is prohibited; form GroupEdit needs updating.
_____________ ERROR collecting webapp/tests/forms/test_member.py ______________
webapp\tests\forms\test_member.py:4: in <module>
    from webapp import forms
webapp\forms.py:58: in <module>
    class GroupEdit(forms.ModelForm):
.venv\lib\site-packages\django\forms\models.py:240: in __new__
    "needs updating." % name
E   django.core.exceptions.ImproperlyConfigured: Creating a ModelForm without either the 'fields' attribute or the 'excl
ude' attribute is prohibited; form GroupEdit needs updating.
!!!!!!!!!!!!!!!!!!! Interrupted: 2 errors during collection !!!!!!!!!!!!!!!!!!!
=========================== 2 error in 7.23 seconds ===========================